awscli/examples/rum/tag-resource.rst

```diff
@@ -0,0 +1,9 @@
+**To assigns one or more tags (key-value pairs) to the specified CloudWatch resource**
+
+The following ``tag-resource`` assigns one or more tags (key-value pairs) to the specified CloudWatch resource. If the command succeeds, no output is returned. ::
+
+	aws rum tag-resource \
```

Comment:
Same here. All of these indents need to be 4 spaces each and not tab indenting.
For more information: https://aws.github.io/aws-cli/docs_styleguide.html#general